sql/marriage/marriages.go

```diff
@@ -151,8 +152,9 @@ func MarriageATXs(db sql.Executor, id ID) ([]types.ATXID, error) {
 	if rows == 0 {
 		return nil, sql.ErrNotFound
 	}
-	sort.Slice(atxs, func(i, j int) bool { return bytes.Compare(atxs[i].Bytes(), atxs[j].Bytes()) < 0 })
-	return slices.Compact(atxs), nil
+	atxIDs := maps.Keys(atxs)
+	sort.Slice(atxIDs, func(i, j int) bool { return bytes.Compare(atxIDs[i].Bytes(), atxIDs[j].Bytes()) < 0 })
+	return atxIDs, nil
```

Comment:
NIT: with Go iterators you could, without relying on the experimental pkg, do:
```suggestion
	return slices.SortedFunc(maps.Keys(atxs), func(a, b types.ATXID) int {
		return bytes.Compare(a.Bytes(), b.Bytes())
	}), nil
```

And change import `"golang.org/x/exp/maps"` to `"maps"`.